lib/experimental.js

```diff
@@ -4,5 +4,5 @@ export {
   defineVocabulary,
   loadDialect, unloadDialect, hasDialect
 } from "./keywords.js";
-export { getSchema, toSchema, canonicalUri, buildSchemaDocument } from "./schema.js";
+export { getSchema, toSchema, canonicalUri, buildSchemaDocument, getAllSchemas } from "./schema.js";
```

Comment:
I don't think this needs to go in the experimental API. You can put it in the stable API (`lib/index.js`) instead.